api/provider/provider.go

```diff
@@ -20,7 +20,7 @@ type Provider interface {
 	BuildCreateTar(app string, src io.Reader, manifest, description string, cache bool) (*structs.Build, error)
 	BuildDelete(app, id string) (*structs.Build, error)
 	BuildGet(app, id string) (*structs.Build, error)
-	BuildList(app string) (structs.Builds, error)
+	BuildList(app string, limit int64) (structs.Builds, error)
```

Comment:
Food for thought, a while back we discussed a ListOptions struct that would have limit and sort properties.